sdk/storage/storage-queue/package.json

```diff
@@ -118,11 +119,17 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-http": "^3.0.0",
+    "@azure/core-auth": "^1.4.0",
+    "@azure/core-client": "^1.6.2",
+    "@azure/core-http-compat": "^2.0.0",
     "@azure/core-paging": "^1.1.1",
+    "@azure/core-rest-pipeline": "^1.10.1",
     "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/core-util": "^1.1.1",
+    "@azure/core-xml": "^1.3.2",
     "@azure/logger": "^1.0.0",
-    "tslib": "^2.2.0"
+    "tslib": "^2.2.0",
+    "uuid": "^8.3.0"
```

Comment:
nitpick: migrate to use `core-util`'s `randomUUID`?

Reply:
might do all of the storage packages at once in a follow-up PR, it's a good idea though!